test/testdata/configs/tagalign.yml

```diff
@@ -0,0 +1,4 @@
+linters-settings:
+  tagalign:
+    auto-sort: true
```

Comment:
Could we add a test when `auto-sort` is `false`?

Reply:
Done. Can be tested by 
```bash
T=tagalign_no_sort.go make test_linters
```